DOCS-4420-4.2-sharing-applications-kt (#1002)
mulesoft/docs-mule-runtime-3: modules/ROOT/pages/sharing-applications.adoc

= Share Applications
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: sharing applications, sharing,

> When working on a team, a Mule runtime engine project increases in size, increases in its number of developers, and must run in other environments, such as test and production.

Reviewer: ```suggestion
When a Mule project involves a whole team, the project increases in size due to the increased number of developers who need to collaborate, and the need to run the project in both test and production environments. However, you can offset this project growth by sharing applications.
```
Please reread and evaluate carefully. I just guessed at the connection between team dev and sharing apps.

> Sharing applications is an opportunity for splitting work across teams, whether these teams work on the same overarching project or on different projects with an accent put on reuse.

Reviewer: See questions about the similar sentence used above.

> Applications are self-contained archives and can therefore be shared via a simple file or web server.

Reviewer: Do we really mean that apps are stored in self-contained archives?
```suggestion
Because applications are stored in self-contained archives, they can be shared via a simple file or web server.
```

> To share applications follow the next steps:

Reviewer: ```suggestion
To share your applications, follow these steps:
```

. Document the name and content of the properties file that each application is expecting to use. +

> The administrator(s) who deploys and runs the Mule application can encrypt the properties file as detailed in xref:secure-configuration-properties.adoc[Secure Configuration Properties]. Administrators must keep the decryption key of the properties file secure.

Reviewer: GLOBALLY replace "(s)" with the more common or likely of the two choices, single or plural: in this case, either "The administrator who deploys and runs" or "The administrators who deploy and run."

[start=2]

> . Generate the file to share the application by the following options :

Reviewer: ```suggestion
. Generate the file used to share the application in one of the following two ways:
```

* Export the application from Anypoint Studio and generate a deployable archive `.jar` file.

> * Package your application using the Mule Maven Plugin and generate a:

Reviewer: ```suggestion
* Package your application into one of the following three types of .jar files using the Mule Maven Plugin:
```

> ** Deployable archive `.jar` file to then deploy it into a running Mule runtime engine.

Reviewer: ```suggestion
** Deployable archive `.jar` file to deploy into a running Mule runtime engine
```

> ** Mule app package `.jar` file to then import it into Studio.

Reviewer: ```suggestion
** Mule app package `.jar` file to import to Studio.
```

> ** Mule app lightweight package `.jar` file to then import it into Studio.

Reviewer: ```suggestion
** Mule app lightweight package `.jar` file to import to Studio.
```

[start=3]

> . After generating the `.jar` file, you can share it with other teams so they can continue working with the application.

Reviewer: ```suggestion
. Share the .jar file with other developers, as appropriate.
```
